tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java

```diff
@@ -161,11 +171,35 @@ public void beginTask( String taskName, int totalWork )
                         {
                             final List<Problem> problems = m.findProblems( location.toFile(), override );
 
-                            problems.addAll( problems );
+                            allProblems.addAll( problems );
                         }
                     }
 
-                    m.reportProblems( allProblems, Migration.DETAIL_LONG, "ide" );
+                    if( allProblems.size() > 0 )
+                    {
+                        CodeMigrationProblems cp = new CodeMigrationProblems();
```

Comment:
no.as I added the param "suffix" to our json file api, for code problems we can pass the project nam as the "suffix",then it would save as one file per one project.
such as "Code-problems-abc-portlet.json" and "code-problems-def-hook.json” if you have two projects need to check.
I don't think all projects' problems be store into one file is a good idea.